autoload/leaderf.vim

```diff
@@ -855,3 +855,9 @@ function! leaderf#setAmbiwidth(val) abort
     exec g:Lf_py "from leaderf.devicons import setAmbiwidth"
     exec g:Lf_py printf("setAmbiwidth('%s')", a:val)
 endfunction
+
+autocmd ColorScheme * call leaderf#highlightDevIcons()
+function! leaderf#highlightDevIcons() abort
+    exec g:Lf_py 'from leaderf.devicons import highlightDevIcons'
+    exec g:Lf_py 'highlightDevIcons()'
```

Comment:
Please add `        if vim.vars.get('Lf_ShowDevIcons', True):` at the beginning of `highlightDevIcons()`.

Reply:
Added.